### ISSUE TITLE
Wrong footer placement on Safari

### ISSUE THREAD

**Issue by tomfran** [tomfran/typo#8]
The footer is not behaving as expected on Safari, the min height of the body seems to be too high.
I tried experimenting with height values with no success. The issue is not present on Chrome. 

Here is an example, you can see the need to scroll on Safari to reveal the footer.

<img width="1514" alt="Screenshot 2024-07-18 at 20 31 30" src="https://github.com/user-attachments/assets/6e0222ef-54e4-4bc4-bb37-fefe9d3fb945">
<img width="1514" alt="Screenshot 2024-07-18 at 20 31 34" src="https://github.com/user-attachments/assets/f1135e49-bc8c-4aaa-a33c-50edca080d7f">

**Comment by runofthemillgeek** [comment]
I can't reproduce this on Safari Version 18.1 (20619.2.8.11.10) using the latest commit (8c8da7151a69c193eb3aa960ad840968fc21485f). Tried switching to light theme too.

<img src="https://github.com/user-attachments/assets/243991c3-9de3-46bc-8e5c-deb38310bcae" width="300px">

**Comment by tomfran** [comment]
Could it be a Safari problem? Do you maybe have other browsers at hand? Nice if it fixed itself

**Comment by runofthemillgeek** [comment]
Not sure, if you're still seeing this issue on a site you have hosted somewhere, drop a link here so that I can check. I checked in Chrome & Firefox and they look good too.

**Comment by runofthemillgeek** [comment]
Yeah, that definitely seems like a quirky thing happening w/ Safari. I couldn't come up with a repro but it seems to go away if you remove `margin-bottom` on `.social-icons`. I'm quite sure it has something to do with `min-height` and `margin` but again, couldn't reproduce it.

But should report this on https://webcompat.com I think.
